system-tests/projects/react18/package.json

```diff
@@ -9,7 +9,8 @@
     "css-loader": "^4.3.0",
     "style-loader": "^2.0.0",
     "vite": "^2.8.0",
-    "webpack": "^4"
+    "webpack": "^4",
+    "webpack-dev-server": "^4"
```

Comment:
now needed since we use webpack 4 in this project. updates on this project/system test will happen at a later time